src/html/find-embeds.js

```diff
@@ -92,13 +92,13 @@ function internalImgEmbed({ type, children }, base, contentext, resourceext) {
 }
 
 function embed(uri, node, whitelist = '', datawhitelist = '', logger) {
-  if ((uri.scheme === 'http' || uri.scheme === 'https') && mm.some(uri.host, datawhitelist.split(','))) {
+  if ((uri.scheme === 'http' || uri.scheme === 'https') && mm.some(uri.host, datawhitelist.split(',').map((s) => s.trim()))) {
```

Comment:
ok, but I would convert the whilelists to an array in the `find` function, as it is needed several times.